go.mod

```diff
@@ -192,20 +195,22 @@ replace (
 	// Ref: https://forum.cosmos.network/t/ibc-security-advisory-dragonberry/7702
 	github.com/confio/ics23/go => github.com/confio/ics23/go v0.9.0
 	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.46.7
+
+	// https://github.com/yihuang/iavl/tree/optim_diff_backport
+	github.com/cosmos/iavl => github.com/yihuang/iavl v0.16.1-0.20230111025924-ae78cc50ad97
 	github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.19-deepcopy-jumptable
 	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.20.2-cronos
 
 	// Fix upstream GHSA-h395-qcrw-5vmq vulnerability.
 	// TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
 	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-
-	// https://github.com/linxGnu/grocksdb/pull/100
-	github.com/linxGnu/grocksdb => github.com/yihuang/grocksdb v1.7.11-0.20230112025116-f41883a71db3
+	github.com/linxGnu/grocksdb => github.com/yihuang/grocksdb v1.7.11-0.20230120153422-b2f936d8375d
```

Comment:
didn't see you submit the HyperClockCache changes to the upstream repo, will you submit it or keep it in `crypto-org-chain` repo?

Reply:
Because it needs a pr in rocksdb itself.

Reply:
https://github.com/facebook/rocksdb/pull/11110
I'll submit grocksdb PR after rocksdb PR merged.

Reply:
don't need the fork anymore, reverted ClockCache for now.